### DISCUSSION TITLE
ORTools - Example2 - Parent Grain Reconstruction in Titanium alloys error

### DISCUSSION THREAD

**Discussion by biginnerMTEX** [mtex-toolbox/mtex#1237] [Ask Anything]
I'm not sure if I can ask this question, but I'm studying about parent grain restructuring using mtex.
In "Choose the method to compound the OR(s): the content around 3 minutes and 40 seconds of YouTube content,

The function 'findpeaks' is not defined for a 'double' input argument.

Error occurred: gaussFit (Line 46)
[peakValue,peakPositionIdx,peakWidth,peakHeight] = findpeaks(y,
'MinPeakDist',classInterval.fine,
'MinPeakHeight',ypos,'Annotate','extents','WidthReference','halfheight');

Error: DefineORs (Line 45)
[misoRange] = gaussFit(classRange,classInterval,counts);

Error: ORTools_example2 (Line 52)
job = defineORs(job);

I keep getting errors like this. I want to get help with how to solve them.

**Comment by kilir** [comment]
Hi,
I guess you need to have ortools https://github.com/ORTools4MTEX/ORTools installed.
Hope that helps.
Tüdiger
